src/main/java/com/researchspace/netfiles/NfsFactory.java

```diff
@@ -89,10 +89,16 @@ public NfsClient getNfsClient(String nfsusername, String nfspassword, NfsFileSys
           fileSystem.getClientOption(NfsFileSystemOption.SFTP_SERVER_PUBLIC_KEY));
     }
     if (NfsClientType.IRODS.equals(clientType)) {
+      int irodsPort;
+      if (StringUtils.isEmpty(fileSystem.getClientOption(NfsFileSystemOption.IRODS_PORT))) {
```

Comment:
Could you maybe use `StringUtils.isBlank(...)` instead? it would catch also when there are blank spaces or tab characters